core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -98,8 +156,11 @@ public ColumnVisibility convert(String value) {
   public String principal = null;
 
   @Parameter(names = "--password", converter = PasswordConverter.class,
-      description = "Enter the connection password", password = true)
-  private Password securePassword = null;
+      description = "conection password (can be specified as '<password>', 'pass:<password>',"
```

Comment:
Should this document `stdin:` prefix?

Reply:
Again I just re-used the original description, with the slight change of omitting the stdin case because I assume it might not work here.

Reply:
Yeah, I wouldn't expect stdin to work here.